### ISSUE TITLE
bug: `lodash/prefer-constant` false positive within class

### ISSUE THREAD

**Issue by gurpreetatwal** [wix/eslint-plugin-lodash#225]
The rule currently warns for methods defined on classes that return constant values, however this isn't the ideal behavior because refactoring those methods to use lodash would produce some weird looking code.

In the following example both methods will be flagged:
```js
class Foo {
  static something() {
    return true;
  }
 
  // other methods 
 
  somethingElse() { 
    return 10;
  }
}
```

the "fixed" code would look like:

```js
class Foo {
 
  // other methods 
}

Foo.something = _.constant(true);
Foo.prototype.something = _.constant(10);
```

which isn't ideal
